apps/web/playwright/fixtures/regularBookings.ts

```diff
@@ -357,5 +362,80 @@ export function createBookingPageFixture(page: Page) {
       await scheduleSuccessfullyPage.waitFor({ state: "visible" });
       await expect(scheduleSuccessfullyPage).toBeVisible();
     },
+    selectTimezoneInSettings: async (name: string, settingsPage: Page) => {
+      await settingsPage.locator("#timezone svg").click();
+      name.includes(" ") && (name = name.replace(" ", "_"));
+      await settingsPage.getByTestId(`select-option-${name}`).click();
+      await settingsPage.getByRole("button", { name: "Update" }).click();
+      await settingsPage.getByRole("button", { name: "Don't update" }).click();
+    },
+    assertRightTimezone: async (
+      timezone: string,
+      bookingSchedulingPage: Page,
+      options?: { inConfirmationPage: boolean; shouldConfirmBooking?: boolean }
+    ) => {
+      if (options?.inConfirmationPage) {
+        await expect(bookingSchedulingPage.getByText("America/New_York")).toBeVisible();
+        if (options.shouldConfirmBooking) {
+          await bookingSchedulingPage.getByTestId("confirm-book-button").click();
+          await expect(bookingSchedulingPage.getByText("This meeting is scheduled")).toBeVisible();
```

Comment:
We should change this to be dynamic and not rely on the exact text. This test will start to fail if the text is changed and that's not ideal.

Reply:
Makes sense! We changed it now 😄